app/models/Case.scala

```diff
@@ -28,3 +28,9 @@ case class Case
   application: Application,
   attachments: Seq[Attachment] = Seq.empty
 )
+
+case class CaseAuditPayload
```

Comment:
Could this not be in a different file rather than sticking it all in the Case model file

Reply:
done